ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-isis-new.md

```diff
@@ -49,7 +49,6 @@ interface Management1
 
 | Interface | Description | Mode | VLANs | Native VLAN | Trunk Group | Channel-Group |
 | --------- | ----------- | ---- | ----- | ----------- | ----------- | ------------- |
-| Ethernet3 | MLAG_PEER_EAPI-LEAF1B_Ethernet3 | *access | *- | *- | *- | 3 |
```

Comment:
The other l2 interfaces are removed from the documentation table. We need fix this.

Reply:
These ones are special because the input model is missing the port-channel interfaces. Since the test scenario for this case is about router isis I don't think we need to handle this. We could even remove the Ethernet3 interfaces, since they have no purpose in this test.

Reply:
It is expected, because the corresponding Port-channel member is not defined.